tests/test_physics.py

```diff
@@ -413,7 +421,7 @@ def test_raycast():
 
```

Comment:
Comment on the line above: `assert raycast_results.hits[0].object_id == 0`

As-is, this code makes the dangerous assumption that the first created object gets id 0. The code is also not very readable. We should fix this line to be `== cube_obj.object_id`.